src/lib/Liblog/pbs_messages.c

```diff
@@ -405,6 +405,8 @@ char *msg_cannot_set_route_que = "Route queues are incompatible with the partiti
 char *msg_queue_not_in_partition = "Queue %s is not part of partition for node";
 char *msg_partition_not_in_queue = "Partition %s is not part of queue for node";
 char *msg_invalid_partion_in_queue = "Invalid partition in queue";
+char *msg_sched_op_not_permitted = "Operation is not permitted on default scheduler";
+char *msg_sched_part_already_used = "Partition is already associated with other scheduler";
```

Comment:
As of today let us say if somebody wants to query a list of partitions that are present we do not have a direct way of 
 knowing them since partition is logical attribute which relates queues/nodes/schedulers. We actually need to query all the queues/nodes/schedulers to get the list of partitions. I think it is better to add this ability first if there is such a requirement for this and then come up with these elaborate messages.  As there are already lot of changes in the same ticket, planning to take both of these as part of another ticket after discussing the these requirement/s in detail.